tests/unit/sanakoyeu_et_al_2018/test_loss.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+
+import pytorch_testing_utils as ptu
+import torch
+from torch import nn
+from torch.nn.functional import binary_cross_entropy_with_logits
+
+import pystiche_papers.sanakoyeu_et_al_2018 as paper
+from pystiche import enc, misc
+
+
+def test_EncodingDiscriminatorOperator_call(subtests, input_image):
+    class TestOperator(paper.EncodingDiscriminatorOperator):
+        def input_enc_to_repr(self, image):
+            return image * 2.0
+
+        def calculate_score(self, input_repr):
```

Comment:
Wouldn't it be better to define this as a function before the class definition and only use it from this method? This way you could reuse the function below.